include/libdnf/comps/group/group.hpp

```diff
@@ -56,18 +60,30 @@ class Group {
     /// If a translation is not found, return untranslated name.
     ///
     /// @replaces dnf:dnf/comps.py:attribute:Group.ui_name
-    std::string get_translated_name() const noexcept { return ""; }
+    std::string get_translated_name(const char * lang);
```

Comment:
Isn't there a way for retrieving a default translation?
In such a case, we should consider keeping the method with no arguments.